CorpusSearch/newdocs

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+#note: this script needs to be run on the manx-search-data repo
+# not the repo it lives in
```

Comment:
because it needs to access the git history of `manx-search-data`

Reply:
It would be useful to have this script stop execution if this condition is not met